WooCommerce/Classes/ViewRelated/CardPresentPayments/CardReaderConnectionController.swift

```diff
@@ -42,6 +42,10 @@ final class CardReaderConnectionController {
         ///
         case connectingFailed(Error)
 
+        /// A mandatory update is being installed
```

Comment:
Maybe, I'm not sure how useful it is to mention that is mandatory here, but optional updates won't happen on the connection controller, so this state will only ever handle mandatory updates. In other words, it might be superfluous but it's accurate 😁